APIs/SentinelHub/Statistical/Examples.qmd

```diff
@@ -6,7 +6,7 @@ The requests below are written in Python. To execute them you need to
 create an OAuth client as is explained
 [here](/APIs/SentinelHub/Overview/Authentication.qmd#python). It is
 named `oauth` in these examples. Jupyter notebook with all examples can
-be downloaded [here]().
+be downloaded [here](https://docs.sentinel-hub.com//api/latest/ba8a238c13144db9ad60bbffef26c241/statistical_api_sh_docs.ipynb).
```

Comment:
that double slash doesn't look right (url gives 404)

that `ba8a238c13144db9ad60bbffef26c241` also doesn't look like a stable long-term reference

Reply:
I'll remove 1 extra slash. 
ba8a238c13144db9ad60bbffef26c241 came directly from the Sentinel Hub documentation: https://docs.sentinel-hub.com/api/latest/api/statistical/examples/ . Then let me not add this link (https://docs.sentinel-hub.com/api/latest/ba8a238c13144db9ad60bbffef26c241/statistical_api_sh_docs.ipynb) to CDSE, and ask Sinergise whether they can provide a stable long term reference. Thanks!